### PR TITLE
Add Timezone Option

### PR DESCRIPTION
**Pull request by stuartstobie**
#### What does this PR do? (please provide any background)

Adds the option to specify a timezone to be used for the database connections. 
Often system time is fine but I'm using it like this to look up data which is time sensitive from different timezones and wanted a way to ensure I was looking stuff up in UTC format.
#### How can be this tested?

Run test.js then add the line `timezone: '0000'` to the configuration within test.js and run it again to make sure it still works.

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.
#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.
#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,9 @@ database.query({
 
 });
 ```
+
+##### Additional Configuration Options
+
+Port, specify the port number you want to use to connect, e.g. `port: 3000`. Will resort to the default port if not used.
```
Comment (stuartstobie) [inline README.md]: This was added a while ago but not documented.

**Comment by matthewjtyas** [comment]
Taking SE/Dev

**Comment by pmcnr-hx** [comment]
Taking SE/Guru.

**Comment by matthewjtyas** [comment]
+1

**Comment by pmcnr-hx** [comment]
:+1:

**Pull request merged**